server/db.js

```diff
@@ -1,5 +1,5 @@
 const mongoose = require("mongoose");
-
+console.log(process.env.MONGO_URI);
```

Comment:
Would you mind removing this logger! Ethan will ask you to remove this

Reply:
Sure.